### PR TITLE
Oded/tfl 975

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/evaluator/config/evaluatorDefaultConfig.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.evaluatorDefaultConfig', []).config(function(ZnkEvaluatorSrvProvider) {
```
Comment (igormas) [inline src/components/evaluator/config/evaluatorDefaultConfig.js]: why it in separate module ?

File: src/components/evaluator/services/znkEvaluatorSrv.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+(function (angular) {
+    angular.module('znk.infra.evaluator').provider('ZnkEvaluatorSrv', function () {
+
+        var _evaluateQuestionGetter;
+
+        this.shouldEvaluateQuestion = function(evaluateQuestionGetter) {
```
Comment (igormas) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: ?

File: src/components/evaluator/services/znkEvaluatorSrv.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            'ngInject';
+
+            var znkEvaluatorSrvApi = {};
+
+            var httpConfig = {
+                timeout: ENV.promiseTimeOut
+            };
+
+            function _shouldEvaluateQuestion() {
+                if(!_evaluateQuestionGetter){
```
Comment (igormas) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: if it not defined then the default one should be executed

File: src/components/evaluator/config/evaluatorDefaultConfig.js (hunk 1 above)
Comment (odeds) [inline src/components/evaluator/config/evaluatorDefaultConfig.js]: that's the default one - it's a different module

File: src/components/evaluator/services/znkEvaluatorSrv.js (hunk 2 above)
Comment (odeds) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: what do you mean?

File: src/components/evaluator/services/znkEvaluatorSrv.js (hunk 3 above)
Comment (odeds) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: the default one is loaded through config phase, didn't want to add dependency in purchaseService

File: src/components/evaluator/services/znkEvaluatorSrv.js (hunk 2 above)
Comment (igormas) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: What this function do ?
What it receive s parameter ?

File: src/components/evaluator/services/znkEvaluatorSrv.js (hunk 3 above)
Comment (igormas) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: U should add, new module for default evaluation controller is too much.
U can skip from adding the purchase module to evaluate module.

Comment (odeds) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: done

File: src/components/evaluator/services/znkEvaluatorSrv.js (hunk 2 above)
Comment (odeds) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: get function to execute and get true or false for evaluate, changed names.

Comment (igormas) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: What does shouldEvaluateQuestion function do ?
What should i provide as evaluateQuestionGetter paramter ?

Comment (odeds) [inline src/components/evaluator/services/znkEvaluatorSrv.js]: register function for evaluate.
changed the name to : shouldEvaluateQuestionFnGetter 
and param to: evaluateQuestionFn

**Pull request merged**